java/quickstart/README.md

```diff
@@ -0,0 +1,158 @@
+
+
+## Costs
+
+This sample uses billable components of Cloud Platform, including:
+
++   Google Cloud Bigtable
+
+Use the [Pricing Calculator][bigtable-pricing] to generate a cost estimate
+based on your projected usage.  New Cloud Platform users might be eligible for
+a [free trial][free-trial].
+
+[bigtable-pricing]: https://cloud.google.com/products/calculator/#id=1eb47664-13a2-4be1-9d16-6722902a7572
+[free-trial]: https://cloud.google.com/free-trial
+
+
+## Before you begin
+
+1. This sample assumes you have [Java 8][java8] installed.
+
+[java8]: http://www.oracle.com/technetwork/java/javase/downloads/
+
+1. Install Maven
+
+These samples use the [Apache Maven][maven] build system. Before getting
+started, be sure to [download][maven-download] and [install][maven-install] it.
+When you use Maven as described here, it will automatically download the needed
+client libraries.
+
+[maven]: https://maven.apache.org
+[maven-download]: https://maven.apache.org/download.cgi
+[maven-install]: https://maven.apache.org/install.html
+
+1. [Select or create][projects] a Cloud Platform project.
+
+1. Enable [billing][billing] for your project.
+
+1. Enable the Cloud Bigtable API][enable_api].
+
+    Note: The quickstart performs an operation on an existing table.
+    If you require your code to create instances or tables,
+    the [Admin API](https://console.cloud.google.com/flows/enableapi?apiid=bigtableadmin.googleapis.com)
+    must be enabled as well.
+
+1. [Set up authentication with a service account][auth] so you can access the API from your local workstation.
+
+1. Follow the instructions in the [user documentation](https://cloud.google.com/bigtable/docs/creating-instance) to
+create a Cloud Bigtable instance (if necessary).
+
+1. Follow the [cbt tutorial](https://cloud.google.com/bigtable/docs/quickstart-cbt) to install the
+cbt command line tool.
+Here are the cbt commands to create a table, column family and add some data:
+```
+   cbt createtable my-table
+   cbt createfamily my-table cf1
+   cbt set my-table r1 cf1:c1=test-value
+```
+
+[projects]: https://console.cloud.google.com/project
+[billing]: https://support.google.com/cloud/answer/6293499#enable-billing
+[enable_api]: https://console.cloud.google.com/flows/enableapi?apiid=bigtable.googleapis.com
+[auth]: https://cloud.google.com/docs/authentication/getting-started
+
+
+## Running the quickstart
+
+The [Quick start](src/main/java/com/example/bigtable/quickstart/Quickstart.java) sample shows a
+basic usage of the Bigtable client library: reading rows from a table.
+
+Build and run the sample using Maven.
+```
+    mvn package
+```
+
+Run the quick start to read the row you just wrote using `cbt`:
+```
+   mvn exec:java -Dexec.mainClass="com.example.cloud.bigtable.quickstart.Quickstart" \
+         -Dexec.args="my-project-id my-bigtable-instance my-table"
+```
+Expected output similar to:
+```
```

Comment:
The expected output will be different here.

Reply:
Done.